anet-tools/src/core/ad.py

```diff
@@ -0,0 +1,56 @@
+import ast
```

Comment:
Why are the examples in package core and the business objects in package examples?